Makefile

```diff
@@ -2,7 +2,8 @@ ARCH := i386
 
 CC      := i686-linux-gnu-gcc-8
 AS      := i686-linux-gnu-as
-CCFLAGS := -g -ffreestanding -Wall -Wextra -Werror -Isrc -lgcc -nostartfiles -fno-pie
+#CCFLAGS := -g -ffreestanding -Wall -Wextra -Werror -Isrc -lgcc -nostartfiles -fno-pie
+CCFLAGS := -g -ffreestanding -Wall -Wextra -Isrc -lgcc -nostartfiles -fno-pie
```

Comment:
This may be just a temp change but -Werror should really be left in. I know it's neat to disable it during development but it should be enabled in general, especially once I get CI up and running.

Reply:
that's why theirs 2 CCFLAGS
#CCFLAGS := -g -ffreestanding -Wall -Wextra -Werror -Isrc -lgcc -nostartfiles -fno-pie(commented out ready to use)
CCFLAGS := -g -ffreestanding -Wall -Wextra -Isrc -lgcc -nostartfiles -fno-pie